codegen/go-composite-schema/main.go

```diff
@@ -50,11 +50,13 @@ go-composite-schema to correctly determine the correct package name.
 
 
   Usage:
-    go-composite-schema [--required] PROPERTY INPUT-FILE OUTPUT-FILE
+    go-composite-schema [--required] [--unexported] PROPERTY INPUT-FILE OUTPUT-FILE
     go-composite-schema -h|--help
     go-composite-schema --version
 
   Options:
+    --required             Implies the given composite schema defines a required property.
+    --unexported           Generate an unexported type for the given composite schema.
```

Comment:
Not that we need to block this PR on this!!!

It's just a nice to have some day when we feel like we have time...